### ISSUE TITLE
Spanish keyboard layout not working

### ISSUE THREAD

**Issue by hferreiro** [andischerer/atom-keyboard-localization#19]
On Linux, using the spanish layout the key to the left of '1' and the key to the left of 'z' return the same keycode 220 corresponding to the former. Additionally there is one map missing: "alted": 91 for keycode "219".

**Comment by andischerer** [comment]
I have added the additinal binding for keycode 219. 
See #12 for multiple keys return the same keyevent.

**Comment by dgaubert** [comment]
On Linux (Ubuntu 14.04 LTS), ES_es layout still doesn't work. ie:
   < resolves to º
   > resolves to ª
   [ resolves to '
   \ resolves to º.

Trying to use _keybindings-generator_ the JSON generated doesn't map properly. Also, _key-down-event_ seems AltGr modifier is not working.

Doing some research, I'm using _de_DE-neo.json_ adding some keybindings:
{
   ...
   "49": {
        "shifted": 33
    },
    "50": {
        "shifted": 34
    },
    "51": {
        "shifted": 183
    },
    "52": {
        "shifted": 36
    },
    "53": {
        "shifted": 37
    },
    "54": {
        "shifted": 38
    },
    "55": {
        "shifted": 47
    },
    "56": {
        "shifted": 40
    },
    "57": {
        "shifted": 41
    },
    ...
}

Hope this helps!

**Comment by andischerer** [comment]
Hmm, the AltGr Key is properly working on my linux box (Ubuntu 15.04).

Could you open chromium with the following url: https://dvcs.w3.org/hg/d4e/raw-file/tip/key-event-test.html.
Click on `Show Options` and check `Old DOM3` in the `Show Fields` section.
Then focus the input box and press the altgr key. Could you take a snapshot from the result ?

**Comment by dgaubert** [comment]
Sorry, I was busy and I forgot reply you, the snapshot:
![atom-keyboard3](https://cloud.githubusercontent.com/assets/1936888/7808576/9080b8e4-0394-11e5-862e-a9e20a90921a.png)

**Comment by andischerer** [comment]
looks OK to me. Seems like the screenshot Shortcut is also captured ;)
Could you try out the new version 1.4. It ships with a new integrated keybindings-generator. 
In the generator AltGr should get highlighted when pressed. Could you confirm that ?

**Comment by ipuustin** [comment]
I'm having the same mapping problem with Swedish/Finnish keyboard layout. The key "left-from-1" and key "left-from-z" appear to both have keycode 220, meaning that I can select which one to use but not both. A better default might be

```
"220": {
    "unshifted": 60,
     "shifted": 62,
     "alted": 124
},
```

because that would mean that I get '<', '>', and '|' keys working. The current mapping in se_SE means that I get only '§' and '½', which are not that important at least from coding point of view.

**Comment by yrjan** [comment]
Same problem for me with the nb_NO (Norwegian) keyboard layout running on Debian testing. I had to remove the remap of keycode 220 to be able to write both '<' and '|'.

**Comment by ismaelgv** [comment]
Same problem here with es_ES:
- < mapped to º
- > mapped to ª

**Comment by adrian-martinez-dev** [comment]
I'm having the same problem with es_LA on ubuntu 14.04 

_<  =  °_
_>  =  |_

**Comment by diegosuarez** [comment]
Same here, and I think it was working properly days ago, it's weird. I changed in the ES_es mapping as suggested

```
"220": {
    "unshifted": 60,
     "shifted": 62,
     "alted": 124
}
```

And works just fine.

**Comment by andischerer** [comment]
should be fixed in v1.4.7
